samples/net/echo_client/overlay-ot.conf

```diff
@@ -34,3 +36,9 @@ CONFIG_NET_CONFIG_PEER_IPV6_ADDR="fdde:ad00:beef::2"
 # Thread by default registers quite a lot addresses.
 CONFIG_NET_IF_UNICAST_IPV6_ADDR_COUNT=6
 CONFIG_NET_IF_MCAST_IPV6_ADDR_COUNT=8
+
+CONFIG_FLASH=y
```

Comment:
I assume this is needed because OpenThread impl stores something in persistent storage. Dunno, but I would add a comment here.

Reply:
Will elaborate on that.